app/components/__tests__/CategoryOverviewView.js

```diff
@@ -1,17 +1,22 @@
 import 'react-native';
 import React from 'react';
+import { Provider } from 'react-redux';
 import renderer from 'react-test-renderer';
+import configureStore from 'redux-mock-store';
 
 import CategoryOverviewView from '../CategoryOverviewView';
 import categories from '../../constants/categories';
 
 it('renders correctly', () => {
+  const store = configureStore()({ products: { list: { records: [] } } });
   const tree = renderer.create(
-    <CategoryOverviewView
-      category={categories[0]}
-      platform="android"
-      onHeaderIconClicked={() => {}}
-    />,
+    <Provider store={store}>
```

Comment:
My suggestion above will simplify such cases. BTW components should know nothing about store.